client/i18n/locales/english/translations.json

```diff
@@ -912,9 +912,10 @@
     "title": {
       "Responsive Web Design": "Responsive Web Design",
       "responsive-web-design": "Responsive Web Design Certification",
-      "JavaScript Algorithms and Data Structures": "JavaScript Algorithms and Data Structures",
-      "javascript-algorithms-and-data-structures": "JavaScript Algorithms and Data Structures Certification",
+      "JavaScript Algorithms and Data Structures": "Legacy JavaScript Algorithms and Data Structures",
+      "javascript-algorithms-and-data-structures": "Legacy JavaScript Algorithms and Data Structures Certification",
       "javascript-algorithms-and-data-structures-v8": "JavaScript Algorithms and Data Structures Certification",
+
```

Comment:
```suggestion
```

Reply:
Sorry, I accidentally added an unnecessary line when resolving the conflict.